examples/ejb-persistent-clustered-timers/helm.yaml

```diff
@@ -0,0 +1,26 @@
+build:
+# TODO - replace with wildfly-extras/master before merging
+  uri: https://github.com/fabiobrz/wildfly-jar-maven-plugin.git
```

Comment:
should be replaced with wildfly-extras and 7.0.1.FInal ref. master ref is changed to snapshot.

Reply:
Ok thanks, fixing the comment here.